### PR TITLE
RxJava interop

### PR DESCRIPTION
**Pull request by arkivanov**

### CODE REVIEW

File: rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package com.badoo.reaktive.rxjavainterop
```
Comment (lukaville) [inline rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt]: probably we should also add interop with rxjava schedulers to reuse the thread pools

Comment (arkivanov) [reply rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt]: Can you tell any use cases?

Comment (lukaville) [reply rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt]: currently, we reuse rxjava 1/2 schedulers so we don't have duplicated thread pools

Comment (arkivanov) [reply rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt]: Sounds reasonable, would it be enough to just convert rxjava2 schedulers to Reaktive. So Reaktive could reuse rxjava2's schedules.

Comment (arkivanov) [reply rxjava-interop/src/main/kotlin/com/badoo/reaktive/rxjavainterop/Completable.kt]: I have pushed the changes, could you please have a look?

**Pull request merged**